functions/string-is-the-same/string-is-same.spec.js

```diff
@@ -0,0 +1,12 @@
+const isSameWord = require("./string-is-same.js");
+
+describe("isTheSameWord", () => {
+  it("should check if one string is equal to another and result in true or false", () => {

```

Comment:
would update the `it` message to should return false when the words are not the same

Reply:
done :)